### PR TITLE
initial add of guided registration

### PR DESCRIPTION
**Pull request by mbay-SAG**
Added our guided registration ui.

### CODE REVIEW

File: Guided-registration/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  <br />
+  <br />
+</p>
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li><a href="#about">About</a></li>
+    <li><a href="#getting started">Getting Started</a></li>
```
Comment (itsyitsy) [inline Guided-registration/README.md:19]:
```suggestion
    <li><a href="#getting-started">Getting Started</a></li>
```
Link is currently broken, I think the anchor cannot contain space character

File: Guided-registration/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (itsyitsy) [inline Guided-registration/LICENSE:1]: You can have that license file additionally but the repo is anyways under Apache 2.0 license. So you don't really need that file here, your choice

**Review by itsyitsy — CHANGES_REQUESTED** [review]
Can you please remove the .DS_Store files?
After that and the small link fix we are ready to merge from my side.

**Comment by mbay-SAG** [comment]
Can be reviewed again. Changed anchor and added .gitignore for .DS_Store as well as correction of url in Dockerfile.

File: Guided-registration/LICENSE (hunk 2 above)
Comment (mbay-SAG) [reply Guided-registration/LICENSE:1]: will keep it in

File: Guided-registration/README.md (hunk 1 above)
Comment (mbay-SAG) [reply Guided-registration/README.md]: corrected

**Review by itsyitsy — APPROVED** [review]
Thank you for contributing @mbay-SAG ! Nice content! 🙂

**Pull request merged**